.eslintrc

```diff
@@ -79,7 +79,7 @@
     "no-extra-boolean-cast": 2,
     "no-irregular-whitespace": 2,
     "no-multi-spaces": 2,
-    "no-else-return": 0,
+    "no-else-return": 2,
```

Comment:
`if (!lol) {
  return;
}

a = 5;
b = a + 5;

c = 5 + 7;`

is better than

`if (lol) {
  a = 5;
  b = a + 5;

  c = 5 + 7;
} else {
  return;
}`